### PR TITLE
Task 6.1

### PR DESCRIPTION
**Pull request by rafazgr**

### CODE REVIEW

File: Task6.1/Author.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿namespace BookCatalogTask
+{
+    public class Author
+    {
+        private string _firstName;
+        private string _lastName;
+
+        public string FirstName
+        {
+            get => _firstName;
```
Comment (IgorKheidorov) [inline Task6.1/Author.cs:12]: try to use const int MAX_LENGTH = 200
init => ValidateAndSetProperty(value, MAX_LENGTH, nameof(FirstName), ref _firstName);

File: Task6.1/Catalog.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
+﻿namespace BookCatalogTask
+{
```
Comment (IgorKheidorov) [inline Task6.1/Catalog.cs:2]: I think that this wrapper class is not necessary, at least now))

File: Task6.1/Book.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                {
+                    throw new ArgumentNullException(nameof(Title), "Title cannot be null.");
+                }
+
+                _title = value;
+            }
+        }
+
+        public DateTime? PublicationDate { get; set; }
+        public List<Author> Authors { get; set; }
```
Comment (IgorKheidorov) [inline Task6.1/Book.cs:23]: Book has to check ISBN by himself, not to let anybody to do this. Think that book with wrong isbn is not a valid object

**Review by IgorKheidorov — APPROVED** [review]
Cool, fix some proposals and can merge

File: Task6.1/Author.cs (hunk 1 above)
Comment (rafazgr) [reply Task6.1/Author.cs:12]: Fixed :)

File: Task6.1/Catalog.cs (hunk 2 above)
Comment (rafazgr) [reply Task6.1/Catalog.cs:2]: Fixed :)

File: Task6.1/Book.cs (hunk 3 above)
Comment (rafazgr) [reply Task6.1/Book.cs:23]: Fixed :)

**Pull request merged**